api/services/Search.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
         qb.orderBy(opts.sort || '', opts.order || 'asc')
       }
 
+      if (opts.is_public) {
+        qb.whereIn('is_public', opts.is_public)
```

Comment:
i dont think whereIn is correct, that is meant for array values, should just be where. but i guess it works!

Reply:
we also used `whereIn` for public posts. I'll try changing both of them to `where`